### PR TITLE
remove out-dated swift-version and support macOS

### PR DESCRIPTION
**Pull request by haifengkao**

### CODE REVIEW

File: .swift-version [hunk 1]
```diff
@@ -1 +0,0 @@
-5.3.2
```
Comment (rex-remind101) [inline .swift-version]: this library is intended to be usable in the swift CLI, specifying a `.swift-version` facilitates that. what reason did you remove this?

Comment (haifengkao) [reply .swift-version]: 截圖 2021-03-18 上午10 09 02

Comment (haifengkao) [reply .swift-version]: If swift cli needs this file, it's should be preserved, but the podspec needs to specify the swift version as well

Comment (rex-remind101) [reply .swift-version]: I found the [link](https://blog.cocoapods.org/CocoaPods-1.7.0-beta/) to the source of what you posted. Thank you for the information. Please restore this file and I will merge the PR.

**Review by rex-remind101 — CHANGES_REQUESTED** [review]
.

File: .swift-version (hunk 1 above)
Comment (rex-remind101) [reply .swift-version]: And publish a new version.

**Comment by haifengkao** [comment]
I have add it back

**Comment by rex-remind101** [comment]
new pod version is published with this change https://cocoapods.org/pods/JSONValueRX

**Pull request merged**